src/common/styles/index.scss

```diff
@@ -1,3 +1,5 @@
 body {
   font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
Do you want to apply `box-sizing: border-box` to all elements or just the body?

If it's to all elements, you can do:
```
* {
  box-sizing: border-box;
}
```

Reply:
Ooh, good call!